docs/src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -21,40 +21,60 @@
 import React from 'react';
 import { cssModules } from 'bpk-react-utils';
 import Helmet from 'react-helmet';
+import BpkContentContainer from 'bpk-component-content-container';
 
 import Heading from '../Heading';
 import IntroBlurb from '../IntroBlurb';
 import MDXContent from '../MDXContent/MDXContent';
 
+import AdditionalLinks from './AdditionalLinks';
 import STYLES from './MarkdownPage.scss';
 
 const getClassName = cssModules(STYLES);
 
 type Props = {
   content: any,
   fileName: string,
-  metaData: {
-    title: string,
-    subtitle: ?string,
-    podLink: ?string,
-    jitPackLink: ?string,
-    npmLink: ?string,
-    ghLink: ?string,
-  },
+  title: ?string,
+  subtitle: ?string,
+  platform: ?$Keys<typeof PLATFORMS>,
+  documentationId: ?string,
+  githubPath: ?string,
```

Comment:
Moved these to the top level for a better API, better not to have things nested if we can avoid it.